gradoop-flink/src/main/java/org/gradoop/flink/model/impl/operators/drilling/Drill.java

```diff
@@ -222,5 +223,16 @@ public DrillUp buildDrillUp() {
       return new DrillUp(
         label, propertyKey, function, newPropertyKey, drillVertex);
     }
+
+    /**
+     * Creates a drill down operation.
+     *
+     * @return drill down operation
+     */
+    public DrillDown buildDrillDown() {
+      Objects.requireNonNull(propertyKey);
```

Comment:
you could move the `Objects.requireNonNull` methods to the setter methods instead of duplicating them in the `build...` methods